ports/windows/Makefile

```diff
@@ -61,7 +61,20 @@ SRC_C = \
 	$(SRC_MOD) \
 	$(wildcard $(VARIANT_DIR)/*.c)
 
+SHARED_SRC_C += $(addprefix shared/,\
+	runtime/gchelper_generic.c \
+	timeutils/timeutils.c \
```

Comment:
Are these 2 files actually needed?  The second one maybe, but the first one?  If it's already working without gchelper_generic then I guess it doesn't need it?

Reply:
Ah yes this would have been a blind copy/paste fail, I can remove those two.